### PR TITLE
MS Build casting error

### PR DESCRIPTION
**Pull request by kerry-Cho**
MSBuild Compile time casting Error

### CODE REVIEW

**Comment by peterjc123** [comment]
cc @soumith @fmassa

File: references/detection/eval.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
+import numpy as np
```
Comment (fmassa) [inline references/detection/eval.py]: this file is completely unrelated to the windows build, right? I think it's a good idea to have it in the repo, but as a separate PR so that we can focus on how to better illustrate this.

**Comment by fmassa** [comment]
I'll send a new PR removing the added file and fixing the lint for C++ in a few minutes. Thanks a lot for the fix on Windows!
